docker-compose.yml

```diff
@@ -6,3 +6,9 @@ services:
     volumes:
       - ./last_send_bug.info:/root/onlyoffice_telegram_bugzilla_notifications/last_send_bug.info
       - ./config.yml:/root/onlyoffice_telegram_bugzilla_notifications/config.yml
+  docspace:
```

Comment:
Kolya, I originall thought about some different design

So right now you just create two instances of same code with little bit different configuration - this increase our cost of maintenance quite a lot for such small service

Maybe we can solve the problem different way

1. Have only 1 instance of code running
2. Inside this instance we have 1 configuration file that have details about:
  a. The channels we need to send the notification
  b. The filter about a bug params
  
  
So in that case we don't need to duplicate anything - just change config look and add additional cycle in main code (loop through channels)

Even the last_send_bug file will be only one

What do you think about it?